sync/src/types.rs

```diff
@@ -43,6 +46,7 @@ const GET_HEADERS_CACHE_SIZE: usize = 10000;
 const GET_HEADERS_TIMEOUT: Duration = Duration::from_secs(15);
 const TX_FILTER_SIZE: usize = 50000;
 const TX_ASKED_SIZE: usize = TX_FILTER_SIZE;
+const ORPHAN_BLOCK_SIZE: usize = 1024;
```

Comment:
I am not sure if the configuration item should be removed here.

Reply:
If we want the sync-parameters configurable, then others like `TX_FILTER_SIZE`/`MAX_ASK_MAP_SIZE` should be put into `SyncConfig` too.